### PR TITLE
Private sourcemaps is a core feature

### PR DESCRIPTION
**Pull request by vmihailenco**

### CODE REVIEW

File: jekyll/_docs/features/private-sourcemaps.md [hunk 1]
```diff
@@ -8,10 +8,8 @@ description: Private sourcemaps
 
 ![](https://s3.amazonaws.com/document-resources/jsbrakeman.png)
 
-**Private sourcemaps are an Advanced feature available on the Business plan and
-above.**
-[Public sourcemaps](https://github.com/airbrake/airbrake-js#source-map)
-are available for all other paid plans.
+Private sourcemaps is a core feature available on all paid plans along with
```
Comment (mmcdaris) [inline jekyll/_docs/features/private-sourcemaps.md]: I think this would read better as "Private sourcemaps are a core feature"

Comment (thompiler) [reply jekyll/_docs/features/private-sourcemaps.md]: (get ready for some pedantry!) :neckbeard: 

Either works. Technically, I think "Private sourcemaps is a core feature" would be fine. The name of the feature itself is not plural and we're not making any use of it as a plural in this sentence. 

eg: "The Beatles is a good band" (technically correct if you're referring to the band as a unit) vs "The Beatles are a good band" 


https://english.stackexchange.com/questions/138238/are-vs-is-for-proper-nouns-which-sound-plural-such-as-band-names

https://forum.wordreference.com/threads/the-beatles-is-the-beatles-are.553835/

Comment (mmcdaris) [reply jekyll/_docs/features/private-sourcemaps.md]: Hahaha, thanks Thom! Excellent explanation, I think "are" flows better for plural sounding nouns.

**Comment by vmihailenco** [comment]
Change `is` to `are` - thanks.

**Pull request merged**